### ISSUE TITLE
I am not able to set httpHeaderField in SDWebImageManager

### ISSUE THREAD

**Issue by SanjeevDew** [SDWebImage/SDWebImage#2838]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by SanjeevDew** [comment]
This issue is persisting in new version 5.0

**Comment by kinarobin** [comment]
You can set httpHeaderField in `SDWebImageDownloader`

[https://github.com/SDWebImage/SDWebImage/blob/24739b21e6efa8f59aa6295da6b9c821c780b227/SDWebImage/Core/SDWebImageDownloader.h#L187](https://github.com/SDWebImage/SDWebImage/blob/24739b21e6efa8f59aa6295da6b9c821c780b227/SDWebImage/Core/SDWebImageDownloader.h#L187)

**Comment by SanjeevDew** [comment]
@kinarobin  Thanks buddy ... much appreciated
